roles/swift-common/handlers/main.yml

```diff
@@ -2,7 +2,6 @@
 - name: pip install swifttool
   pip: name=/opt/stack/{{ swift.swifttool_dir  }}
        extra_args="-i {{ openstack.pypi_mirror | default(omit) }}"
-       virtualenv={{ swift.venv_dir }}
```

Comment:
we should not change how ubuntu and centos will work (which is still install swfit tool in venv).  Only for openstack_distro_type == 'osp' we should install swifttool at system level.